lighthouse-core/test/audits/performance-budget-test.js

```diff
@@ -23,14 +23,15 @@ describe('Performance: Resource budgets audit', () => {
           {url: 'http://third-party.com/file.jpg', resourceType: 'Image', transferSize: 70},
         ]),
       },
-      URL: {requestedURL: 'http://example.com', finalURL: 'http://example.com'},
+      URL: {requestedUrl: 'http://example.com', finalURL: 'http://example.com'},
     };
     context = {computedCache: new Map(), settings: {}};
   });
 
   describe('with a budget.json', () => {
     beforeEach(() => {
       context.settings.budgets = [{
+        path: '/',
```

Comment:
this isn't required from users but we need it here because we expect the budget to have already gone through our config, correct?

Reply:
Yep, the budgets in this test skip validation, so it's necessary to add the "/".